src/renderer/fetch-types.ts

```diff
@@ -42,18 +41,10 @@ export async function fetchTypeDefinitions(version: string): Promise<string> {
  *
  * @param version
  */
-export async function removeTypeDefsForVersion(version: string) {
-  const fs = await fancyImport<typeof fsType>('fs-extra');
+export function removeTypeDefsForVersion(version: string): Promise<void> {
   const _version = normalizeVersion(version);
   const typeDefsDir = path.dirname(getOfflineTypeDefinitionPath(_version));
-
-  if (fs.existsSync(typeDefsDir)) {
-    try {
-      await fs.remove(typeDefsDir);
-    } catch (error) {
-      throw error;
-    }
-  }
+  return fs.remove(typeDefsDir);
```

Comment:
Small explanation of this refactor: fs-extra's remove() is a no-op if the path doesn't exist, so fs.existsSync() is not needed. And since we rethrow any error we catch, the try-catch block is not useful.